translations/nl-be.json

```diff
@@ -1159,6 +1159,7 @@
   "also-found-in": "Ook gevonden in",
   "uploaded-in-thing": "Geüpload in {thing}",
   "creation-date": "Aanmaakdatum",
+  "download-agenda-documents-info": "Vink de ministers aan wier documenten je wilt downloaden. Documenten zonder minister worden altijd toegevoegd aan het bestand.",
```

Comment:
We always grab the docs without mandatees (which should be all the announcements and the "verslag").
Hypothetically, if someone downloads all documents per mandatee they will always have these so results in many duplicates across zips (there also will be duplicates when multiple mandatees are on 1 agendaitem but that is unavoidable).

Do we want a select box for all documents without mandatees that is checked by default. Could be a follow up PR
Perhaps even a filter on the `agendaItemType`

Reply:
I don't disagree, but this comes verbatim from the FP: https://www.notion.so/Feature-Passports-Kaleidos-4e5ca9a91ae24b609ee2f56b2166f370?p=645bcc0f0d544e0c961d6eb3a64d815e&pm=c

You could ask there/ask Hannes directly and we can add it as a follow up ticket?

Reply:
Follow up PR is fine. Just thinking ahead. We could have waited for user feedback on the issue.